docs/modules/ROOT/pages/configure-cmos.adoc

```diff
@@ -22,21 +22,36 @@ By default, logs can be either provided to standard output `(stdout)` or to serv
 If file output is used for the logs, these files will appear in the `/logs/` directory.
 Further, if desired, these logs can externally be shared via a volume or bind mount.
 
-To mount files into a container, use the `docker -v <source>:<destination>` command, ideally as read-only.
+To mount files into a container, use the `docker -v <source>:<destination>` command.
 For further information, refer to the official documentation:
 - https://docs.docker.com/storage/volumes/
 - https://docs.docker.com/storage/bind-mounts/
 
 == Cluster Monitor
 
-The cluster manager service of the CMOS stack will auto-start and self-configure itself to monitor a Couchbase cluster using the configuration supplied through the following environment variables:
+The cluster monitor exposes a REST API endpoint from the container so that it can be used externally to add/remove Couchbase clusters.
 
-- `CLUSTER_MONITOR_USER=${CLUSTER_MONITOR_USER:-admin}`: The username for the Couchbase cluster to monitor.
-- `CLUSTER_MONITOR_PWD=${CLUSTER_MONITOR_PWD:-password}`: The password for the Couchbase cluster to monitor.
-- `CLUSTER_MONITOR_ENDPOINT=${CLUSTER_MONITOR_ENDPOINT:-http://localhost:7196}`: The endpoint of the Couchbase cluster to monitor.
+The cluster monitor can be configured via a bespoke REST API: `curl -u "${CLUSTER_MONITOR_USER}:${CLUSTER_MONITOR_PWD}" -X POST -d '{ "user": "'"${COUCHBASE_USER}"'", "password": "'"${COUCHBASE_PWD}"'", "host": "'"${COUCHBASE_ENDPOINT}"'" }' "${CLUSTER_MONITOR_ENDPOINT}/api/v1/clusters"`
 
-The cluster manager  exposes a REST API endpoint from the container so that it can be used externally to add/remove Couchbase clusters. 
-The cluster monitor is also capable of running any scripts found in `/etc/healthcheck/` folder to perform health checks on the Couchbase cluster.
+- `CLUSTER_MONITOR_USER=${CLUSTER_MONITOR_USER:-admin}`: The username for the cluster monitor tool.
+- `CLUSTER_MONITOR_PWD=${CLUSTER_MONITOR_PWD:-password}`: The password for the cluster monitor tool.
+- `CLUSTER_MONITOR_ENDPOINT=${CLUSTER_MONITOR_ENDPOINT:-http://localhost:7196}`: The endpoint of the cluster monitor tool.
+- `COUCHBASE_USER=${COUCHBASE_USER:-Administrator}`: The username for accesing the Couchbase cluster.
+- `COUCHBASE_PWD=${COUCHBASE_PWD:-password`: The password for accesing the Couchbase cluster.
+- `COUCHBASE_ENDPOINT=${COUCHBASE_PWD:-http://db2:8091`: This should be set to a node that you want to monitor in a Couchbase cluster.
```

Comment:
Not sure `COUCHBASE_PWD` should be here - also the syntax on some of these is incorrect (missing right `}`)